js/acquia_lift_target.js

```diff
@@ -226,6 +226,11 @@ Drupal.acquiaLiftLearn = (function() {
     initializeSession();
     settings = Drupal.settings.acquia_lift_learn;
     api = Drupal.acquiaLiftV2API.getInstance();
+    for (var agent_name in Drupal.settings.acquia_lift_target.agent_map) {
+      if (Drupal.settings.acquia_lift_target.agent_map.hasOwnProperty(agent_name)) {
+        site_name_prefixes[agent_name] = Drupal.settings.acquia_lift_target.agent_map[agent_name].site_name_prefix;
```

Comment:
I realize I am probably nit-picking a temporary approach or didn't understand an impediment here - are we able to just use the variable Drupal.settings.acquia_lift.customer_site at JavaScript level? Then we can get rid of all the updates in file "acquia_lift.module", and prefix "SITE_NAME-" or empty string to the decision id only at file "acquia_lift_target.js".